Suggested new UI

anishsarangi opened the pull request:
@vbanos This PR contains the suggested changes by @mekarpeles for the new UI of the extension. This will look much like this 
<img width="197" alt="screen shot 2018-12-19 at 1 31 35 am" src="https://user-images.githubusercontent.com/15776213/50179776-ed3b2a00-032d-11e9-9c4f-a510c1793104.png">

Please review it. Thank you!!

vbanos commented:
You don't need to set `width: 100%` for buttons in the CSS! Please use `btn-block`
http://getbootstrap.com/docs/3.3/css/#buttons

anishsarangi commented:
@vbanos I have updated the PR by doing the following 
1)Used btn-block instead of width:100% .
2)Used bootstrap-material-deisign-icons npm package for feedback glyphicon.
3)Changed the color of the Context button to make it stand out from other buttons

Thank you.
